ui/app/pages/create-account/connect-hardware/connect-screen.js

```diff
@@ -1,206 +1,226 @@
-        },
-      },
-      steps.map((step) => (
-        h('div.hw-connect', {}, [
-          h('h3.hw-connect__title', {}, step.title),
-          h('p.hw-connect__msg', {}, step.message),
-          h('img.hw-connect__step-asset', { src: `images/${step.asset}.svg`, ...step.dimensions }),
-        ])
-      ))
-      )
+  scrollToTutorial = () => {
+    if (this.referenceNode) {
+      this.referenceNode.scrollIntoView({behavior: 'smooth'})
     }
+  }
 
-    renderFooter () {
-      return (
-        h('div.hw-connect__footer', {}, [
-          h('h3.hw-connect__footer__title', {}, this.context.t('readyToConnect')),
-          this.renderButtons(),
-          h('p.hw-connect__footer__msg', {}, [
-            this.context.t('havingTroubleConnecting'),
-            h('a.hw-connect__footer__link', {
-              href: 'https://support.metamask.io/',
-              target: '_blank',
-            }, this.context.t('getHelp')),
-          ]),
-        ])
-      )
-    }
+  renderLearnMore () {
+    return (
+      <p className="hw-connect__learn-more" onClick={this.scrollToTutorial}>
+        {this.context.t('learnMore')}
+        <img className="hw-connect__learn-more__arrow" src="images/caret-right.svg" alt="" />
+      </p>
+    )
+  }
 
-    renderConnectScreen () {
-      return (
-        h('div.new-account-connect-form', {}, [
-          this.renderHeader(),
-          this.renderButtons(),
-          this.renderTrezorAffiliateLink(),
-          this.renderLearnMore(),
-          this.renderTutorialSteps(),
-          this.renderFooter(),
-        ])
-      )
-    }
+  renderTutorialSteps () {
+    const steps = [
+      {
+        asset: 'hardware-wallet-step-1',
+        dimensions: {width: '225px', height: '75px'},
+        title: this.context.t('step1HardwareWallet'),
+        message: this.context.t('step1HardwareWalletMsg'),
+      },
+      {
+        asset: 'hardware-wallet-step-2',
+        dimensions: {width: '300px', height: '100px'},
+        title: this.context.t('step2HardwareWallet'),
+        message: this.context.t('step2HardwareWalletMsg'),
+      },
+      {
+        asset: 'hardware-wallet-step-3',
+        dimensions: {width: '120px', height: '90px'},
+        title: this.context.t('step3HardwareWallet'),
+        message: this.context.t('step3HardwareWalletMsg'),
+      },
+    ]
 
-    render () {
-      if (this.props.browserSupported) {
-        return this.renderConnectScreen()
-      }
-      return this.renderUnsupportedBrowser()
-    }
-}
+    return (
+      <div
+        className="hw-tutorial"
+        ref={node => {
+          this.referenceNode = node
+        }}>
+        {steps.map((step, index) => (
+          <div className="hw-connect" key={index}>
+            <h3 className="hw-connect__title">{step.title}</h3>
+            <p className="hw-connect__msg">{step.message}</p>
+            <img className="hw-connect__step-asset" src={`images/${step.asset}.svg`}{...step.dimensions} alt="" />
```

Comment:
Nit: There should probably be a space between the `src` attribute and the spread attributes